.github/workflows/mutation.yml

```diff
@@ -49,10 +51,22 @@ jobs:
             It requires private repo read/write permissions." >> $GITHUB_STEP_SUMMARY
             exit 1
           fi
+
+      - name: Add label to block auto merge
+        if: github.event.workflow_run.event == 'pull_request' && steps.download-artifacts.outputs.found_artifact == 'true'
+        run: |
+          curl \
+            -X POST \
+            -H "Authorization: Bearer ${{ secrets.MUTATION_TOKEN }}" \
+            -H "Accept: application/vnd.github.v3+json" \
+            "https://api.github.com/repos/${{ github.repository }}/issues/${{ github.event.workflow_run.pull_requests[0].number }}/labels" \
+            -d '["🚧 pr/do-not-merge"]'
```

Comment:
I think we should use a new label to make sure the intent is clear. I think people would be hesitant to remove a  `do-not-merge` label. Maybe something like `pr/review-mutation`. With a new label we can have a description that explains what to do.

The new label would need to be in the mergify config as well.

Reply:
Yes, this is something I thought about as well. Do we have an emoji for this? xD

Reply:
Not sure, here are the first ones that came to mind:
- ❗ or ❕
- ⚠️ or ⚠︎

Feel free to pick 😄